### DISCUSSION TITLE
GraphQl, Apollo Server and Type-GraphQL.

### DISCUSSION THREAD

**Discussion by IsaacBighi** [MichalLytek/type-graphql#1660] [Q&A] [answered]
Hello, I tried to make an application using GraphQL with Apollo server and type-GraphQL.
When I installed GraphQl and the Apollo server, Apollo asked for the latest version of Graph, when I updated, the error was in type-graphql, asking to downgrade the GraphQl version.
I wanted to know if there is a way around this or if I'm just a layman and couldn't find the answer, LOL.

**Comment by MichalLytek** [comment, marked as answer]
Install latest beta release of TypeGraphQL 2.0.

**Comment by andreseloysv** [comment]
Hello IsaacBighi,

We would like to invite you to join our discussion regarding the 2.0 release. Your insights would be particularly valuable, especially if you have concerns about installing a beta release. Please feel free to share your thoughts and feedback with us at [TypeGraphQL Discussions](https://github.com/MichalLytek/type-graphql/discussions/1544).

Looking forward to your participation!
